target/linux/ramips/image/mt7621.mk

```diff
@@ -140,6 +140,33 @@ define Device/alfa-network_quad-e4g
 endef
 TARGET_DEVICES += alfa-network_quad-e4g
 
+define Device/ampedwireless_ally
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  UBINIZE_OPTS := -E 5
+  FILESYSTEMS := squashfs ubifs
+  KERNEL_SIZE := 4096k
+  IMAGE_SIZE := 32768k
+  KERNEL_INITRAMFS := $(KERNEL_DTB) | custom-initramfs-uimage flashable-initramfs |\
+     edimax-header -s CSYS -m RN68 -f 0x001c0000 -S 0x01100000
+  IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
+  DEVICE_VENDOR := Amped Wireless
```

Comment:
Please order the variables more like the other devices.

Reply:
Honestly I don't see a lot of consistency throughout the mt7621.mk, but I gave it a shot.

Reply:
My main point is that DEVICE_VENDOR should be at the top, not at the bottom....

Reply:
ok thanks, I think this should be better